CONTRIBUTING.md

```diff
@@ -27,3 +27,47 @@ We are happy to see your enthusiasm. This table lists the criteria and how impor
 | Include core TF API aka no contrib   | Yes | Optional |
 | Include README     | Yes | Yes |
 | Maintain version dependency updates     | Yes | Optional |
+
+
+## Expectations
+
+### Contributors
+
+- Please read the Python Dev Guide and in particular the [Google Python Style Guide](https://github.com/google/styleguide/blob/gh-pages/pyguide.md).
+
+- Your Pull Request should be small and include a single logical change.
+
+- Smaller changes are easier to review and evaluate, and in general smaller self-contained PRs make it easier to debug or, if needed, rollback.
+
+- If your code appears to include generic functionality, make sure you have not reinvented the wheel.
+
+- Check other repos and, for instance, prefer adapting code that already exists over a new implementation.
+
+- Your change should pass existing unit tests.
+
+- You should provide a sufficiently detailed description of your Pull Request.
+
+- Good descriptions make it easy to understand what the goal of the change is. Writing a good description may also make it apparent if multiple changes are being bundled in the same Pull Request that could be split.
+
+- The LGTM from the code owner ("code review") should ensure both the change intention and implementation are suitable for the codebase being changed. The review from Google team can then focus on the specificities of the Python language.
+
+- Your Pull Request should not have any pending commits (modified in Local Branch but not uploaded).
+
+- If you made changes to your Pull Request before sending it to for approval, make sure you uploaded it, too.
```

Comment:
nit: This line and the line above seem unnecessary to me, but I'm good with them staying in too. :)

Reply:
Agree